src/schema/v2/me/userInterestsConnection.ts

```diff
@@ -0,0 +1,58 @@
+import { GraphQLFieldConfig, GraphQLInt } from "graphql"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { pageable } from "relay-cursor-paging"
+import { ResolverContext } from "types/graphql"
+import { paginationResolver } from "../fields/pagination"
+import {
+  UserInterestConnection,
+  userInterestCategoryEnum,
+  userInterestInterestTypeEnum,
+} from "../userInterests"
+
+export const UserInterestsConnection: GraphQLFieldConfig<
+  void,
+  ResolverContext
+> = {
+  type: UserInterestConnection,
+  args: pageable({
+    category: {
+      type: userInterestCategoryEnum,
+      description:
+        "UserInterest category to select. 'collected_before' or 'interested_in_collecting' category",
+    },
+    interestType: {
+      type: userInterestInterestTypeEnum,
+      description:
+        "UserInterest InterestType to select. 'Artist' or 'Gene' type",
+    },
+    page: { type: GraphQLInt },
```

Comment:
Bc we tend to use the connection pagination arguments, while we can offer `page` and `size` support as well, we might wait to add those until they're needed?

Reply:
Good point. I guess, we don't need them for now. I'll remove `page` and `size` 👍